### PR TITLE
7alip | Homework 1 Week 2

### PR DESCRIPTION
**Pull request by 7alip**

### CODE REVIEW

File: 7alip_week1/05.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+// 5.3 Declare another variable a that has the value of z but rounded to the nearest integer.
+let a = Math.round(z);
+
+// 5.4 console.log a.
+console.log(a); // 7
+
+// 5.5 So now we have z and a find a way to compare the two values and store the highest of the two in a new variable.
+let highestVal;
+if (a > z) {
```
Comment (marcianoviereck) [inline 7alip_week1/05.js:18]: Good! You can also use [math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max). But this is only a suggestion

File: 7alip_week1/08.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+// 8.4 Now use typeof to log the actual type of your variables.
+console.log(typeof myString); // string
+console.log(typeof myBool); // boolean
+console.log(typeof myObject); // object
+console.log(typeof myArray); // object
+
+// 8.5 Now compare the types of your different variables with one another.
+// 8.6 Make sure to also show a message when the variables you are comparing are not the same type.
+const compare = (a, b) => {
```
Comment (marcianoviereck) [inline 7alip_week1/08.js:35]: wow, nice solution!

**Review by marcianoviereck — APPROVED** [review]
Nicely done! And very cool solutions

**Comment by 7alip** [comment]
Merged week 3 homework

**Comment by wouterkleijn** [comment]
Thanks @7alip for submitting your homework and @marcianoviereck for reviewing.

**Pull request closed without merging**